packages/algob/src/builtin-tasks/test.ts

```diff
@@ -1,33 +1,32 @@
-import glob from "glob";
-import path from "path";
+import * as fs from "fs";
+import Mocha from "mocha";
 
-import { internalTask } from "../internal/core/config/config-env";
-import { cmpStr } from "../lib/comparators";
-import type { AlgobRuntimeEnv, ResolvedAlgobConfig } from "../types";
-import { TaskArgs } from "./deploy";
-import { TASK_TEST_GET_TEST_FILES } from "./task-names";
+import { task } from "../internal/core/config/config-env";
+import { testsDirectory } from "../lib/script-checkpoints";
+import type { AlgobConfig } from "../types";
+import { loadFilenames } from "./deploy";
+import { TASK_TEST } from "./task-names";
 
-export default function (): void {
-  internalTask(TASK_TEST_GET_TEST_FILES)
-    .addOptionalVariadicPositionalParam(
-      "filenames",
-      "An optional list of files to test",
-      []
-    )
-    .setAction(runTests);
-}
-
-function loadFiles (fileNames: string[], config: ResolvedAlgobConfig): string[] {
-  if (fileNames.length !== 0) {
-    return fileNames;
+async function runTests (config: AlgobConfig): Promise<void> {
+  if (!fs.existsSync(testsDirectory)) {
+    console.log("Test directory doesn't exists or exists with a different name. Please ensure that it's name is 'test'.");
```

Comment:
```suggestion
    console.log("Test directory does not exist or exists with a different name. Please ensure that it's name is 'test'.");
```

Reply:
I have completely removed this line. I have added a builtin error now which will be thrown by `loadfiles` func.